src/frontend/src/lib/stores/modal.store.ts

```diff
@@ -97,6 +99,7 @@ const initModalStore = <T>(): ModalStore<T> => {
 		openReceiveBitcoin: () => set({ type: 'receive-bitcoin' }),
 		openAboutWhat: () => set({ type: 'about-what' }),
 		openAboutHow: () => set({ type: 'about-how' }),
+		openDappDetails: () => set({ type: 'dapp-details' }),
```

Comment:
Shouldn't opening the modal also receive the data as parameter - i.e. the dapp description?

Reply:
Oh I like this, did not catch this pattern. Trying to figure out how to get the data in the modal if it's passed like this.